d2app/app.go

```diff
@@ -626,9 +626,11 @@ func (a *App) ToCreateGame(filePath string, connType d2clientconnectiontype.Clie
 
 // ToCharacterSelect forces the game to transition to the Character Select (load character) screen
 func (a *App) ToCharacterSelect(connType d2clientconnectiontype.ClientConnectionType, connHost string) {
-	// https://github.com/OpenDiablo2/OpenDiablo2/issues/790
-	characterSelect := d2gamescreen.CreateCharacterSelect(a, a.asset, a.renderer, a.inputManager,
+	characterSelect, err := d2gamescreen.CreateCharacterSelect(a, a.asset, a.renderer, a.inputManager,
 		a.audio, a.ui, connType, connHost)
+	if err != nil {
+		fmt.Printf("unable to create character select screen: %s", err)
+	}
 
 	a.screen.SetNextScreen(characterSelect)
 }
```

Comment:
I'm wondering if we should just bubble up the error all the way to `d2app.App` so that the app goes to the error screen. What do you think?